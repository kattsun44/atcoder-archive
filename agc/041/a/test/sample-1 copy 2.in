584324828731963967 17679858894948916 79688006791148582
