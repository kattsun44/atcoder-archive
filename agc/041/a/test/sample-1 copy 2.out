31004073948099833
